dupeChecker.py

```diff
@@ -8,17 +8,65 @@
 import argparse
 import ConfigParser
 
+
 BASE_PATH = os.path.dirname(os.path.realpath(__file__))
 
+class FileHeuristicCache:
+    def __init__(self, fn):
+        """
+        Construct an object that stores file comparison heuristics.
+        :param filename of file
+        """
+        self.fn = fn
+        self.hash = self.getHash()
+
+
+    def __eq__(self, other):
+        return self.hash == other.hash
```

Comment:
Should add a TODO here to handle the case the hash method returns a 'None'...we shouldn't consider those files duplicates.